ydb/public/lib/ydb_cli/common/csv_parser.cpp

```diff
@@ -152,7 +155,7 @@ class TCsvToYdbConverter {
             Builder.TzTimestamp(token);
             break;
         default:
-            TMisuseException() << "Unsupported primitive type: " << Parser.GetPrimitive();
+            throw TMisuseException() << "Unsupported primitive type: " << Parser.GetPrimitive();
```

Comment:
```suggestion
            ythrow TMisuseException() << "Unsupported primitive type: " << Parser.GetPrimitive();
```

Maybe we want to have file + line? text of error message is not unique

Reply:
There are only `throw` statements everywhere in the file